model/models.py

```diff
@@ -128,3 +130,87 @@ def forward(self, sub, rel):
 
 		score = torch.sigmoid(x)
 		return score
+
+	# Documentation
+
+
+class CompGCN_CTKGC(CompGCNBase):
+		'''
+		Implements the CTKGC Scoring Function as in the paper https://link.springer.com/article/10.1007/s10489-021-02438-8 and writes to the console the total Parameter uesd in the Network
+
+		The scoring Function works by
+			1. Build the Entity-relation matrix by multiplying sub_emb and rel_emb.transpose(1,0)
```

Comment:
Shouldnt it be (e^s)^T x (e^r), as in section 3.2 Fusion layer in the paper? So the subject emebedding needs to eb transposed, not the relation embedding

Reply:
yes true